### PR TITLE
devco patch(l10n_co): update l10n_co to IFRS rules and clean up

### PR DESCRIPTION
**Pull request by blaggacao**
**Description of the issue/feature this PR addresses:**
Update of the l10n_co in order to make it IFRS ready. Clean up (remove) misleading and incompliant tax entries

**Current behavior before PR:**
The l10n_co has not been updated in a while and at the time of it's creation (by me), we did not have yet all the information necessary to produce reliable data. After gaining an indepth insight into colombian accounting matters, and after several legislative changes during the last months, we decided to deprecate the old version and come up with a major rework. Tax entries gave the impression of false completeness, we had to acknowledge during the learning curve, that tax configuration is a lot more complex, than we could imagine at first. In the old configuration this can provoke false security for users and ultimately a bad impression fo the product (Odoo), as it apparently makes an implicit promise here, which it cannot hold.

Just one example of a "relatively" simple tax: IVA can have three different modes: "deducible", "mayorvalor", y "promediado" depending on the characteristics of the company and the product. This is just ignored in the current tax setup.

**Desired behavior after PR is merged:**
We have a well formed CoA according to IFRS and a reliable base for further configuration and customization throughout the Colomban Odoo Ecosystem.
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

Signed-off-by: David Arnold dar@devco.co

@qdp-odoo : On behalf o f DevCO Colombia we would quite like to update l10n_co to the current situation and newest knowledge and criteria, that we gained throughout the learning curve. I'm aware, that this can possibly be a lengthy process, so I'd like to get the discussion and feedback rounds started.

cc/ @fpodoo because you effectively merged the previous version.

cc/ @kurkop

### CODE REVIEW

**Comment by kurkop** [comment]
@blaggacao Can you add the Odoo CLA signature as DevCO organitazion?

**Comment by blaggacao** [comment]
https://github.com/odoo/odoo/pull/11755 y
https://github.com/odoo/odoo/pull/11754

**Comment by mart-e** [comment]
Thanks for the important update. Do you plan to add the taxes again in the future or would it be too complex?
@nhomar I believe you have already worked on this CoA. Could you give your opinion on a functional point of view?

File: addons/l10n_co/__openerp__.py [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
     'depends': [
         'account',
         'base_vat',
     ],
     'data': [
-        'data/account.account.tag.csv',
-        'data/account_chart_template_before.xml',
-        'data/account.account.template.csv',
+        'data/data_account_type.xml',
+        'data/account_account_template.xml',
```
Comment (mart-e) [inline addons/l10n_co/__openerp__.py]: this file include references to `chart_template_niif` while it is the file `account_chart_template.xml` that creates it and is loaded after.

**Comment by mart-e** [comment]
By the way this change breaks the enterprise module `l10n_co_reports` so cc @pvy-odoo

**Comment by blaggacao** [comment]
@mart-e Adding back the taxes is just too complex, we have developed around 25 key modules to get Colombian accounting right, not just "seemingly right" and ar now in the endeavor of happy implementing with happy and astonished clients :dancer: 

We also have a module l10n_co_reports, we could possibly feedback once we are in partnership mode (application pending with Alonso in the US).

Please beware, that there is being carried out a series of tax reforms to modernize the Colombian fiscal system. We commit on behalf of our clients to keep things up to date and feed back appropriate parts upstream shortly after we solve them.

File: addons/l10n_co/__openerp__.py (hunk 1 above)
Comment (blaggacao) [inline addons/l10n_co/__openerp__.py]: true. fixed.

**Comment by blaggacao** [comment]
@mart-e If possible we would like to give this a significant push. It's the reason we actually negate new clients to migrate from community to enterprise-co, at the moment. It's just a very big mess....

**Pull request closed without merging**